lib/metadataTypes/MetadataType.js

```diff
@@ -340,18 +340,15 @@ class MetadataType {
             const metadataResults = createResults
                 .concat(updateResults)
                 .filter((r) => r !== undefined && r !== null)
-                .map((r) => r.body.Results)
+                .map((r) => r.Results)
```

Comment:
was looking into this one as well when I tested. you sure there are no cases WITH `body.` ?

Reply:
in theory this impacts a lot:
![image](https://user-images.githubusercontent.com/1917227/160272650-f28ddab8-008f-4d8a-a28c-1b1224cfb1e3.png)

Reply:
the comment did say something about soap only but I wonder if your new SDK is doing the same for REST and SOAP here?